examples/foundational/26d-gemini-multimodal-live-text.py

```diff
@@ -53,10 +56,14 @@ async def main():
         llm = GeminiMultimodalLiveLLMService(
             api_key=os.getenv("GOOGLE_API_KEY"),
             # system_instruction="Talk like a pirate."
+            transcribe_user_audio=True,
+            transcribe_model_audio=True,
         )
-        llm.set_model_only_text()  # This forces model to produce text only responses
+        llm.set_model_modalities(
+            GeminiMultimodalModalities.TEXT
+        )  # This forces model to produce text only responses
 
-        tts = CartesiaMultiLingualTTSService(api_key=os.getenv("CARTESIA_API_KEY"))
+        tts = CartesiaTTSService(api_key=os.getenv("CARTESIA_API_KEY"))
```

Comment:
```suggestion
        tts = CartesiaTTSService(api_key=os.getenv("CARTESIA_API_KEY"), voice_id="79a125e8-cd45-4c13-8a67-188112f4dd22")
```

Reply:
thanks for the quick PR @imsakg. Cartesia tts just needs a `voice_id`, otherwise 👍